openedx/core/djangoapps/user_api/views.py

```diff
@@ -138,7 +138,7 @@ def post(self, request):
                 "username": accounts.USERNAME_CONFLICT_MSG.format(username=username),
             }
             errors = {
-                field: [{"user_message": conflict_messages[field]}]
+                field: [conflict_messages[field]]
```

Comment:
Since it's a core change, let's leave a comment saying something like `# PhilU changes` at the end of the line?
If you decide to do it, please do the same at line `160` and line `168`.